shell/client/setup-wizard/wizard.html

```diff
@@ -100,6 +100,7 @@
 <template name="setupWizardHelpFooter">
 <div class="setup-help-row">
   <span class="setup-help-label">Need help?</span>
+  <a class="setup-help-link" target="_blank" href="/admin/status">System Log</a>
```

Comment:
Marginally weird that this is visible even if you're not authenticated, but I'm not convinced it's worth hiding.